src/bin/js/missing_primitives.js

```diff
@@ -0,0 +1,31 @@
+// This files aims to give missing primitives to js_of_ocaml compiler
+// the missing primitives are replace with dummies
+
+//Provides: unix_times
+//Requires: unix_gettimeofday
+function unix_times () {
+	var utime = unix_gettimeofday ();
+	return BLOCK(0, utime, utime, utime, utime)
+}
+//Provides: unix_setitimer
+function unix_setitimer () {
+  return BLOCK(0, 0, 0, 0)
+}
```

Comment:
I think we'd like to have a real `setitimer` implementation to be able to have timeouts in the js version, if it's reasonably possible ?

Reply:
I agree, i don't know if it's possible but i will look into it

Reply:
Setting a timeout will be possible at top level see the `main_js.ml` file.